google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -137,6 +137,17 @@ def visit_empty_set_expr(self, type_):
             _type_map_inv[type(type_[0])]
         )
 
+    def visit_like_op_binary(self, binary, operator, **kw):
+        """Build a LIKE clause."""
+        if binary.modifiers.get("escape", None):
+            raise NotImplementedError("ESCAPE keyword is not supported by Spanner")
+
+        # TODO: use ternary here, not "and"/ "or"
+        return "%s LIKE %s" % (
+            binary.left._compiler_dispatch(self, **kw),
+            binary.right._compiler_dispatch(self, **kw),
+        )
```

Comment:
In this method I've erased the part with addind `ESCAPE` substring and added an exception raise in case `ESCAPE` is used.